applications/TrilinosApplication/custom_strategies/convergencecriterias/trilinos_up_criteria.h

```diff
@@ -202,9 +202,16 @@ class TrilinosUPCriteria : public ConvergenceCriteria< TSparseSpace, TDenseSpace
 
```

Comment:
man we should finally unify the ConvCrits (this one does basically the same as the DisplacementCriteria) and remove it, it is really not nicely implemented

Reply:
100% Agree